Formula/s/spglib.rb

```diff
@@ -0,0 +1,62 @@
+class Spglib < Formula
+  desc "C library for finding and handling crystal symmetries"
+  homepage "https://spglib.readthedocs.io/"
+  url "https://github.com/spglib/spglib/archive/refs/tags/v2.1.0.tar.gz"
+  sha256 "31bca273a1bc54e1cff4058eebe7c0a35d5f9b489579e84667d8e005c73dcc13"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "gcc" # for gfortran
+
+  def install
+    common_args = %w[
+      -DSPGLIB_WITH_Fortran=ON
+    ]
+    [true, false].each do |shared_lib_type|
+      build_dir = "build_#{shared_lib_type ? "shared" : "static"}"
+      system "cmake",
+             "-S", ".",
+             "-B", build_dir,
+             *common_args,
+             "-DSPGLIB_SHARED_LIBS=#{shared_lib_type ? "ON" : "OFF"}",
+             *std_cmake_args
+      system "cmake", "--build", build_dir
+      system "cmake", "--install", build_dir
+    end
```

Comment:
Can you just add this twice? It's much easier to read than the loop.

Reply:
Yes, but wouldn't it be a bit more cumbersome to maintain? Also in the case of the test?

Reply:
Not really, since it would be much clearer what is happening.